extractor.go

```diff
@@ -190,20 +190,20 @@ func firstFile(tr *tar.Reader) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func formationFromProcfile(p procfile.Procfile) (Formation, error) {
+func formationFromProcfile(app *App, p procfile.Procfile) (Formation, error) {
 	switch p := p.(type) {
 	case procfile.StandardProcfile:
-		return formationFromStandardProcfile(p)
+		return formationFromStandardProcfile(app, p)
```

Comment:
I've been referring to the `map[string]string` procfile format as "standard", but I could switch to "legacy". If I switch it, I'd want to do it across the board. What do you think?